db/sdg.rb

```diff
@@ -23,3 +23,5 @@
 ].each do |code|
   SDG::Target.where(code: code, goal: SDG::Goal.find_by!(code: code.split(".").first)).first_or_create!
 end
+
+3.times { |kind| SDG::Phase.where(kind: kind).first_or_create! }
```

Comment:
What about looping over `SDG::Phase.kinds.values`? By doing this way, if someone customizes the kinds array definition for any reason won't have to update this line too.

```
SDG::Phase.kinds.values.each { |kind| SDG::Phase.where(kind: kind).first_or_create! }
```

Reply:
Done!

Reply:
Updated the `dev_seeds` as well :+1:.